types/discord-rich-presence/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Type definitions for discord-rich-presence 0.0
+// Project: https://github.com/devsnek/discord-rich-presence#readme
+// Definitions by: Sv443 <https://github.com/Sv443>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { EventEmitter } from "events";
+
+declare namespace createClient {
+    /**
+     * Go to the `Rich Presence > Visualizer` settings of your [Discord application](https://discord.com/developers/applications) to find out how this would be rendered.
+     */
+    interface PresenceInfo {
+        [key: string]: string | number | boolean;
+
+        /** The user's current party status */
+        state: string;
+        /** What the player is currently doing */
+        details: string;
+        /** Epoch seconds for game start - setting this will show the time as "elapsed" */
+        startTimestamp: number;
+        /** Epoch seconds until game's end - setting this will show the time as "remaining" */
+        endTimestamp: number;
```

Comment:
Those can be a Date as well, am I correct?
https://github.com/discordjs/RPC/blob/master/src/client.js#L490-L495
```suggestion
        /** Epoch seconds for game start - setting this will show the time as "elapsed" */
        startTimestamp: Date | number;
        /** Epoch seconds until game's end - setting this will show the time as "remaining" */
        endTimestamp: Date | number;
```

Reply:
It's an Epoch timestamp so the Date object can't be used here

Reply:
Good observation! I'll change it